src/js/chat/handlers.js

```diff
@@ -422,12 +420,6 @@ var privmsg = exports.privmsg = function(channel, data) {
         store.displayNames[data.from] = data.tags['display-name'];
     }
 
-    // filter blacklisted emojis
-    blacklistedEmoji.forEach(function(emoji) {
```

Comment:
k.. well then this may be fine.. idc really since v7 will handle this all better by filtering usable ones